library/std/src/os/windows/io/handle.rs

```diff
@@ -190,14 +190,14 @@ impl BorrowedHandle<'_> {
     /// object as the existing `BorrowedHandle` instance.
     #[stable(feature = "io_safety", since = "1.63.0")]
     pub fn try_clone_to_owned(&self) -> crate::io::Result<OwnedHandle> {
-        self.duplicate(0, false, c::DUPLICATE_SAME_ACCESS)
+        self.duplicate(0, false, sys::c::DUPLICATE_SAME_ACCESS)
     }
 
     pub(crate) fn duplicate(
         &self,
-        access: c::DWORD,
+        access: u32,
         inherit: bool,
-        options: c::DWORD,
+        options: u32,
```

Comment:
is it part of the Windows API definitions that DWORD means 32 bits even on AArch64?

Reply:
See [Windows data types](https://learn.microsoft.com/en-us/windows/win32/winprog/windows-data-types). At this point they've ossified to the point they'll never be changed and are documented as such. The official Rust bindings don't even bother with the alias any more, even internally.

Reply:
Mind you, Rust actually has fixed integer types `u32` whereas C89 did not.

Reply:
Cool, then I'm satisfied.